cmd/local-volume-provisioner/main.go

```diff
@@ -111,9 +113,17 @@ func main() {
 }
 
 func getNode(client *kubernetes.Clientset, name string) *v1.Node {
-	node, err := client.CoreV1().Nodes().Get(name, metav1.GetOptions{})
-	if err != nil {
-		klog.Fatalf("Could not get node information: %v", err)
+	var retries int
+
+	for {
+		node, err := client.CoreV1().Nodes().Get(name, metav1.GetOptions{})
+		if err == nil {
+			return node
+		}
+
+		retries++
```

Comment:
add a log here and one-second sleep, then the user knows what happens, e.g.

```
klog.Infof("Could not get node information (remaining retries: %d): %v", maxGetNodesRetries - retries, err)
time.Sleep(time.Second)
```

Reply:
a small sleep ensures that the provisioner will not fail in a short time if we want it to tolerate network failures.